packages/ui/scripts/script.js

```diff
@@ -0,0 +1,47 @@
+// packages/ui/script.js
```

Comment:
Let's add detailed JSDoc to this and let others know what this exactly does why we needed etc?

Reply:
I'll remove this, it's not required